### PR TITLE
Support marking day by week

### PR DESCRIPTION
**Pull request by Moreno97**
Supports marking a day of the week on each month
See #40

### CODE REVIEW

File: src/calendar/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React, {Component} from 'react';
-import {View, ViewPropTypes} from 'react-native';
+import React, { Component } from 'react';
```
Comment (Inbal-Tish) [inline src/calendar/index.js]: Please remove formatting changes

**Comment by Inbal-Tish** [comment]
@Moreno97 Hi. Please remove all formatting changes so I can review. Thanks

**Comment by Moreno97** [comment]
Sure @Inbal-Tish , sorry for the delay, didn't see the notification.

**Comment by AntonOmelchuk** [comment]
when will it merge?

**Comment by castalonirenz** [comment]
is it already merged?

**Comment by tramhotps** [comment]
please merge it to the source code

**Comment by Inbal-Tish** [comment]
@Moreno97  If this PR still relevant please address my comments and reopen it.

**Pull request closed without merging**